docs/writing-docs/mdx.md

```diff
@@ -2,51 +2,33 @@
 title: 'MDX'
```

Comment:
This diff was collapsed by default, so hoping a comment here will help reviewers not miss it. 😅